bigtable-hbase-1.x-parent/bigtable-hbase-1.x-mapreduce/pom.xml

```diff
@@ -198,18 +206,11 @@ limitations under the License.
                 <targetDependency>org.apache.hbase:hbase-server:${hbase1.version}</targetDependency>
               </targetDependencies>
               <ignoredDependencies>
-                <!-- Unfortunately we can't use hadoop's version of slf4j
-                because dropwizard metrics slf4j logger requires a higher version
-                we can't shade it either -->
-                <dependency>org.slf4j:slf4j-api</dependency>
-                <!-- TODO: figure out what to do about metrics mismatch -->
-                <dependency>io.dropwizard.metrics:metrics-core</dependency>
-
-                <!-- BEGIN: force upgrade transitive deps due to security reports -->
-                <dependency>org.apache.commons:commons-compress</dependency>
-                <dependency>org.tukaani:xz</dependency>
-                <dependency>commons-codec:commons-codec</dependency>
-                <!-- END: force upgrade transitive deps due to security reports -->
+                <!-- TODO: figure out why this diverges from hbase-client -->
+                <ignoredDependency>org.slf4j:slf4j-api</ignoredDependency>
+                <!-- For some reason, hbase-server pulls in test deps -->
```

Comment:
can we exclude?

Reply:
we already do. Those deps are marked as `provided` and will not be included in the jar. So we just ignore the difference here